.tx/config

```diff
@@ -17,6 +17,14 @@ type         = QT
 minimum_perc = 23
 lang_map     = pt: pt_PT, zh: zh_CN
 
+[o:sledgehammer999:p:qbittorrent:r:qbittorrent_webui_public]
+file_filter  = src/webui/www/public/lang/<lang>.json
+source_file  = src/webui/www/public/lang/en.json
```

Comment:
```
[o:sledgehammer999:p:qbittorrent:r:qbittorrent_webui_json]
file_filter  = src/webui/www/transifex/<lang>.json
source_file  = src/webui/www/transifex/en.json
```

Reply:
>`qbittorrent_webui_json`

isn't it a new resource on transifex? Shouldn't I remove this whole section instead?

> source_file  = src/webui/www/transifex/en.json

Do I also need to change the path that the i18next-parser emits? (in the config file `i18next-parser.config.mjs`)

Reply:
Done.